File: basex/php/read/basex_read.php
#! /usr/bin/php
<?php
// ------------------------------------------------------------------
//	read/basex_read.php
//
//					Oct/27/2011
//
// ------------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);
//
include "xml_manipulate.php";
include "text_manipulate.php";
include "curl_get.php";
// ------------------------------------------------------------------
print "*** 開始 ***\n";


$url = 'http://admin:admin@localhost:8984/rest/cities?query=/';

$xml_string = curl_get_proc ($url);

$dict_aa = xml_to_dict_proc ($xml_string);

dict_display_proc ($dict_aa);
// print	$xml_string;

print	"*** 終了 ***\n";
// ------------------------------------------------------------------
?>

File: basex/python/read/basex_read.py
#! /usr/bin/python
# -*- coding: utf-8 -*-
#
#
#	read/basex_read.py
#
#					May/31/2012
#
# ------------------------------------------------------------------
import sys
reload(sys)
sys.setdefaultencoding('utf-8')
#
sys.path.append ("/var/www/data_base/common/python_common")
from curl_get import curl_get_proc
from text_manipulate import dict_display_proc
from xml_manipulate import xml_to_dict_proc
#
# ------------------------------------------------------------------
print ("*** 開始 ***")
	
url_in = 'http://admin:admin@localhost:8984/rest/cities?query=/'
#
str_aa = curl_get_proc (url_in)
#
dict_aa=xml_to_dict_proc (str_aa)
dict_display_proc (dict_aa)
#
print ("*** 終了 ***")
# ------------------------------------------------------------------
